From 84b5131730ec2addc539387cc07f4409ac84a1d5 Mon Sep 17 00:00:00 2001
From: Carsten Haitzler (Raster <raster@rasterman.com>
Date: Sun, 2 Nov 2008 11:47:42 +1100
Subject: [PATCH] glibc: disable patch that moves ld.so.cache to /var/run to not run ldconfig

ldconfig is run at boot as a work around for ld.so.cache being in volatile
ramdisk. this means it is not persistent and its caching won't help on boot.
disable this.
---
 packages/glibc/glibc_2.6.1.bb |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/packages/glibc/glibc_2.6.1.bb b/packages/glibc/glibc_2.6.1.bb
index ab343ea..c1a9598 100644
--- a/packages/glibc/glibc_2.6.1.bb
+++ b/packages/glibc/glibc_2.6.1.bb
@@ -41,7 +41,6 @@ SRC_URI = "\
   file://arm-longlong.patch;patch=1 \
   file://fhs-linux-paths.patch;patch=1 \
   file://dl-cache-libcmp.patch;patch=1 \
-  file://ldsocache-varrun.patch;patch=1 \
   file://nptl-crosscompile.patch;patch=1 \
   file://glibc-2.5-local-dynamic-resolvconf.patch;patch=1;pnum=0 \
   file://glibc-check_pf.patch;patch=1;pnum=0 \
@@ -61,6 +60,8 @@ SRC_URI = "\
   file://march-i686.patch;patch=1;pnum=0 \
 "
 
+#  file://ldsocache-varrun.patch;patch=1 \
+
 # Build fails on sh3 and sh4 without additional patches
 SRC_URI_append_sh3 = " file://no-z-defs.patch;patch=1"
 SRC_URI_append_sh4 = " file://no-z-defs.patch;patch=1"
-- 
1.5.4.3

